integration_tests/Makefile

```diff
@@ -80,15 +85,20 @@ run-horaedb-cluster: build-horaedb
 	nohup ${HORAEDB_BINARY_PATH} --config ${HORAEDB_CONFIG_FILE_1} > ${CLUSTER_HORAEDB_STDOUT_FILE_1} 2>&1 &
 	sleep 30
 
-run: prepare build-meta
-	$(HORAEDB_TEST_BINARY)
+run:
+	make run-local
+	make run-cluster
+	make run-compaction-offload
```

Comment:
In the previous implementation, it just run `prepare` to clear the old process in the very beginning, which may lead to `ConnectError` bug. (e.g. `run-local` and `run-cluster` use the same grpc port but the previous process is not stop yet, which makes the later process can not use the port.)

The current implementation avoids the bug describe above by running `prepare` before each (different) processes.

It needs further improvement in the future -- The log now looks a bit strange